src/targets/gpu/mlir.cpp

```diff
@@ -196,7 +196,6 @@ struct mlir_program
 
     MlirType make_tensor(const shape& s) const
     {
-        assert(s.standard());
```

Comment:
This `assert` shouldnt be removed as we dont pass strides to the tensor and MLIR assumes standard shape.